All Doc Changes for v0.25.0 Release (#556)
redwoodjs/redwoodjs.com: docs/deploy.md

The most important Redwood configuration is to set the `apiProxyPath` in your `redwood.toml` This sets the API path for your serverless functions specific to your hosting provider.

### 2. Build Command
The build command is used to prepare the Web and API for deployment. Additionally, other actions can be run during build such as database migrations. The Redwood build command must specify one of the supported hosting providers (aka `target`):

```shell
yarn rw prisma deploy <target>
```

For example:

```shell
# Build command for Netlify deploy target

> yarn rw prisma deploy netlify

Reviewer: ```suggestion
yarn rw deploy netlify
```

```

```shell
# Build command for Vercel deploy target

> yarn rw prisma deploy vercel

Reviewer: ```suggestion
yarn rw deploy vercel
```

```

```shell
# Build command for Serverless deploy target

> yarn rw prisma deploy serverless

Reviewer: ```suggestion
yarn rw deploy serverless
```

> # Build command for Serverless deploy target
> yarn rw prisma deploy serverless

Author: ```suggestion
# Build command for AWS Lambdas using the https://serverless.com framework
rw deploy aws serverless --side api
```

```

### 3. Prisma and Database
Redwood uses Prisma for managing database access and migrations. The settings in `api/prisma/schema.prisma` must include the correct deployment database, e.g. postgresql, and the database connection string.

To use PostgreSQL in production, include this in your `schema.prisma`:

```javascript
datasource DS {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

The `url` setting above accesses the database connection string via an environment variable, `DATABASE_URL`. Using env vars is the recommended method for both ease of development process as well as security best practices.

> 

Author: ```suggestion

Whenever you make changes to your `schema.prisma`, you must run the following command:
```shell	
$ yarn rw prisma migrate dev # creates and applies a new Prisma DB migration	
```	

_Don't forget to save, commit, and push any changes prior to deploying!_
```

}
```

The `url` setting above accesses the database connection string via an environment variable, `DATABASE_URL`. Using env vars is the recommended method for both ease of development process as well as security best practices.

> Whenever you make changes to your `schema.prisma`, you must run the following commands:

Reviewer: Shouldn't we still have this and just say you need to run yarn rw prisma migrate dev instead?
